tests/security/check_kernel_config/dm_crypt.pm

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: dm crypt -> add flags to optionally bypass kcryptd
+#          workqueues, the options are 'no_read_workqueue' and
+#          'no_write_workqueue'
+#
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#88873, tc#1768663
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
```

Comment:
Looks like you only use **reconnect_mgmt_console** from **utils**, am I right?
Perhaps you can try `use utils 'reconnect_mgmt_console';`
But it is fine without changing if keeping this way too. :)

Reply:
Thanks, then let us keep it